docs/local_install_skaffold.md

```diff
@@ -1,51 +1,280 @@
+
+#### [1. Kubernetes](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+##### Mac
+
+First of all, download the `Docker desktop` installer from https://www.docker.com/products/docker-desktop. You'll have to create an account there to do so. Then run it to install Docker on your machine.
 
-This guide will help you run the Substra platform on your machine in development mode, with a 2 organisations setup.
+Once installed, launch Docker and open its "preferences" panel. In the Kubernetes tab, check the `Enable Kubernetes` checkbox.
+
+Kubernetes will take a while to launch the first time, but once it is done, you can move on to configuring.
+
+If you don't have `homebrew` installed, please install it first with the following command:
+
+```sh
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+##### Ubuntu
 
-- [Setup](#setup)
-  - [1. Get the source code](#1-get-the-source-code)
-  - [2. Install Docker and Kubernetes](#2-install-docker-and-kubernetes)
-  - [3. Configure Kubernetes](#3-configure-kubernetes)
-  - [4. Install skaffold](#4-install-skaffold)
-- [Running the platform](#running-the-platform)
-- [Login, password and urls](#login-password-and-urls)
+Please use **[Kubernetes v1.15](https://kubernetes.io/docs/tasks/tools/install-kubectl/)** (client & server):
 
-## Setup
+```sh
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.15.0/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+kubectl version --client
+```
+
+#### [2. Minikube](https://minikube.sigs.k8s.io/docs/start/)
+
+```sh
+# Mac
+brew install minikube
+
+# Ubuntu
+# Get the executable & install it (Please use the up-to-date version)
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_1.6.2.deb && sudo dpkg -i minikube_1.6.2.deb
+```
+
+#### [3. Helm](https://helm.sh/)
+
+V3 is not supported yet, please use [Helm v2.16.1](https://github.com/helm/helm/releases/tag/v2.16.1)
+
+```sh
+# Mac
+# Get the executable
+curl -LO https://get.helm.sh/helm-v2.16.1-darwin-amd64.tar.gz
+# Extract the downloaded archive
+tar xzvf helm-v2.16.1-darwin-amd64.tar.gz
+cd darwin-amd64/
+# Move the executables to your local bin
+sudo mv tiller helm /usr/local/bin/
+
+# Linux (amd64)
+# Get the executable
+curl -LO https://get.helm.sh/helm-v2.16.1-linux-amd64.tar.gz
+# Extract the downloaded archive
+tar xzvf helm-v2.16.1-linux-amd64.tar.gz
+cd linux-amd64/
+# Move the executables to your local bin
+sudo mv tiller helm /usr/local/bin/
+```
+
+#### [4. Skaffold](https://skaffold.dev/)
+
+```sh
+# Mac
+brew install skaffold
+# Or get the executable
+curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-darwin-amd64
+# Make it executable on your machine
+chmod +x skaffold
+# Move it to your local bin
+sudo mv skaffold /usr/local/bin
```

Comment:
Since we tell the user to have homebrew installed, we could just use the brew command and not specify an alternate one.